src/Controls/tests/DeviceTests/Memory/MemoryTests.cs

```diff
@@ -46,6 +48,12 @@ public async Task HandlerDoesNotLeak(Type type)
 	{
 		SetupBuilder();
 
+#if ANDROID
+		// NOTE: skip certain controls on older Android devices
+		if (type == typeof (DatePicker) && !OperatingSystem.IsAndroidVersionAtLeast(30))
+				return;
```

Comment:
This is working for me locally on an API 23 emulator -- so I don't think it is really leaking.

We skipped like this in another place:

https://github.com/dotnet/maui/blob/9fcccd57b5a3d664e4788b3c7fc3edc10010beaf/src/Controls/tests/DeviceTests/Elements/NavigationPage/NavigationPageTests.cs#L302C53-L303

Since this would still run on the newer Android emulators, this seems fine for now. We are more interested in iOS on this PR.